src/soko_game.rs

```diff
@@ -12,15 +12,15 @@ use ratatui::{
 };
 
 pub fn view(model: &mut Model, frame: &mut Frame) {
-    let game_window = &mut model.game.window;
+    let game_window = &mut model.soko_game;
 
     let main_area = frame.area();
 
     let [left_area, right_area] =
         Layout::horizontal([Constraint::Percentage(50), Constraint::Percentage(50)])
             .areas(main_area);
 
-    let title = &game_window.world.name;
+    let title = &game_window.current_state.name;
```

Comment:
calling this a current state seems smart? i think typically its bad form to write var names as types, and "world" just sounds like the name of a type if you ask me.